data/TreeData.js

```diff
@@ -1,125 +1,112 @@
-const configData = {
-  rootId: 'topic',
-  items: {
-    topic: {
-      id: 'topic',
-      type: 'radio',
-      isVisible: true,
-      data: {
-        title: 'My topic',
-        type: 'layerGroup',
-        isExpanded: true,
-      },
-      children: ['baselayer1', 'sublayers1', 'sublayers2'],
-    },
-    baselayer1: {
-      id: 'baselayer1',
-      type: 'checkbox',
-      isVisible: true,
-      isBaseLayer: true,
-      data: {
-        title: 'OSM Baselayer',
-        type: 'xyz',
-        url: 'https://c.tile.openstreetmap.org/{z}/{x}/{y}.png',
-      },
-    },
-    sublayers1: {
-      id: 'sublayers1',
-      isVisible: true,
-      type: 'radio',
-      data: {
-        title: 'Others layers',
-        type: 'layerGroup',
-        isExpanded: true,
-      },
-      children: ['countryBorders', 'usaPop'],
-    },
-    countryBorders: {
-      id: 'countryBorders',
-      type: 'checkbox',
-      isVisible: false,
-      data: {
-        title: 'Countries Borders',
-        styleId: 'countryBorders',
-        type: 'vectorLayer',
-        url:
-          'https://openlayers.org/en/latest/examples/data/geojson/' +
-          'countries.geojson',
-      },
+import Style from 'ol/style/Style';
+import Circle from 'ol/style/Circle';
+import Stroke from 'ol/style/Stroke';
+import Fill from 'ol/style/Fill';
+
+export default {
+  baselayer1: {
+    name: 'OSM Baselayer',
+    visible: true,
+    isBaseLayer: true,
+    data: {
+      type: 'xyz',
+      url: 'https://c.tile.openstreetmap.org/{z}/{x}/{y}.png',
     },
-    usaPop: {
-      id: 'usaPop',
-      type: 'checkbox',
-      isVisible: true,
-      data: {
-        title: 'USA Population Density',
-        styleId: 'usaPop',
-        type: 'wmts',
-        url:
-          'https://services.arcgisonline.com/arcgis/rest/services/' +
-          'Demographics/USA_Population_Density/MapServer/WMTS/?layer=0' +
-          '&style=default&tilematrixset=EPSG%3A3857&Service=WMTS&' +
-          'Request=GetTile&Version=1.0.0&Format=image%2Fpng&',
-        projection: 'EPSG:3857',
+  },
+  sublayers1: {
+    name: 'Others layers',
+    visible: true,
+    type: 'parent',
+    children: {
+      countryBorders: {
```

Comment:
Sorry I just see that fxinf the conflict. Why not using an array ? since 'countryBorders' value is never used.

Reply:
I mean using an array for the children property.

Reply:
I was thinking the same when I saw that the `items` property is an object. I wanted to be consistent but I totally agree when you say we should switch to an array for `items` and `children`.